sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -1,15 +1,32 @@
-## Release History
+# Release History
 
-### 0.1.0b4 (unreleased)
+## 0.1.0b5 (2022-07-06)
 
-#### Features Added
+### Features Added
 
-#### Breaking Changes
+- Allow Input/Output objects to be used by CommandComponent.
+- Added MoonCake cloud support.
+- Unified inputs/outputs building and validation logic in BaseNode.
+- Allow Git repo URLs to be used as code for jobs and components.
+- Updated AutoML YAML schema to use InputSchema.
+- Added end_time to job schedule.
+- MIR and pipeline job now support registry assets.
 
-#### Bugs Fixed
+### Breaking Changes
 
-#### Other Changes
+### Bugs Fixed
 
+- Have mldesigner use argparser to parse incoming args.
+- Bumped pyjwt version to <3.0.0.
+- Reverted "upload support for symlinks".
+- Error message improvement when a YAML UnionField fails to match.
+- Reintroduced support for symlinks when uploading.
+- Hard coded registry base URL to eastus region to support preview.
+
+## 0.1.0b4 (unreleased)
```

Comment:
Could we backfill the release date for b4? I think it was 2022-06-16

Reply:
Added this in.